src/lib.rs

```diff
@@ -1201,6 +1202,21 @@ impl Hasher {
     }
 }
 
+impl hash::Hasher for Hasher {
+    fn finish(&self) -> u64 {
+        let mut bytes = [0; 8];
+
+        self.finalize_xof().fill(&mut bytes);
+
+        u64::from_be_bytes(bytes)
```

Comment:
Since BLAKE3 is inherently little-endian, it would make more sense to use `u64::from_le_bytes(bytes)` here; on most common processors (which are all little-endian), it's also faster (which is why BLAKE3 is little-endian in the first place).